### PR TITLE
added snippet view

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -22,7 +27,7 @@ export function setNetworkLayer() {
   let options = {};
 
   // Access Token
-  const authToken = '';
+  const authToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI1ODZkZDIxZWJiYjYzMTAwMWQzNDdlYzMiLCJuYW1lIjoiRGFuIEtpbSIsImVtYWlsIjoiaGNraW0wNjI1QGdtYWlsLmNvbSIsImlhdCI6MTQ4MzU5MjIyMn0.WAJhwzUqUqXMEYLLQ7eqaZ32SOKZwjXxsQxInslaU7g';
```
Comment (ace0625) [inline app/app.js]: Delete the token.

File: app/shared/styles.js [hunk 2]
```diff
@@ -77,4 +77,12 @@ const styles = StyleSheet.create({
   }
 });
 
+// font family names:
```
Comment (ace0625) [inline app/shared/styles.js:122]: Is this for later use?

Comment (ochanje210) [reply app/shared/styles.js:122]: I added custom fonts via Xcode and in order to use them correctly, their exact names must be known without a single typo. So I left them here as reference.

Comment (ace0625) [reply app/shared/styles.js:122]: Alright. Good to know.

**Comment by ace0625** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/recbook/recbook-client/pull-request/22/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=recbook/recbook-client)

**Pull request merged**